contents/docs/session-replay/_snippets/android-privacy.mdx

```diff
@@ -0,0 +1,24 @@
+> 🚧 **NOTE:** Currently, this isn't supported if using Jetpack Compose views. We're investigating this [issue](https://github.com/PostHog/posthog-android/issues/158).
+
+Your replays may contain sensitive information. For example, if you're building a banking app you may not want to capture how much money a user has in their account. 
```

Comment:
I dont think we need this intro anymore, since the main page already has an intro. Same for the other platforms